components/Header.tsx

```diff
@@ -1,7 +1,7 @@
 'use client'
 
-import Pic from '../assets/logo_black.png'
-import Pic2 from '../assets/logo_white.png'
+// import Pic from '../assets/logo_black.png'
+// import Pic2 from '../assets/logo_white.png'
```

Comment:
@DengreSarthak , you cannot simply remove an import without caring about where these imports are being used. Look at lines 27 and 35. Those lines are not going to work.

Let's discuss further in Discord.